### ISSUE TITLE
test_invalid_glob has started failing on windows (CI)

### ISSUE THREAD

**Issue by declension** [quodlibet/quodlibet#4021]
Steps to reproduce
------------------
Run a recent PR build on CI

Expected Output
---------------
No failures on Windows

Actual Output
-------------

```
tests\test_util_cover.py:111: in test_invalid_glob
    os.path.abspath(self._find_cover(self.song).name), f)
E   AttributeError: 'NoneType' object has no attribute 'name'
```
e.g. 
* https://github.com/quodlibet/quodlibet/runs/6951286577?check_suite_focus=true
* https://github.com/quodlibet/quodlibet/runs/6956307536?check_suite_focus=true

Test System
-----------
msys2 on Github CI

**Comment by declension** [comment]
Also https://github.com/quodlibet/quodlibet/runs/6956137431?check_suite_focus=true

**Comment by declension** [comment]
Python 3.10 more than Windows, it turned out

Fixed by #4027
